search/utils.py

```diff
@@ -0,0 +1,366 @@
+"""
+Search functions.
+"""
+
+from __future__ import unicode_literals
+
+from collections import defaultdict
+from lxml import etree
+import logging
+
+from django.conf import settings
+from elasticsearch.helpers import bulk
+from elasticsearch.exceptions import NotFoundError
+from elasticsearch_dsl.connections import connections
+from elasticsearch_dsl import Search, Mapping, query
+
+from statsd.defaults.django import statsd
+
+from learningresources.models import get_preview_url, LearningResource
+from search.search_indexes import get_course_metadata
+
+log = logging.getLogger(__name__)
+
+DOC_TYPE = "learningresource"
+INDEX_NAME = settings.HAYSTACK_CONNECTIONS["default"]["INDEX_NAME"]
+URL = settings.HAYSTACK_CONNECTIONS["default"]["URL"]
+CONN = connections.create_connection(hosts=[URL])
+PAGE_LENGTH = 10
```

Comment:
I'm completely open to suggestions. It's only there because 10 is the actual Elasticsearch default, so when doing pagination I needed it in a variable.

If we change the value, it won't stop the default result from Elasticsearch (we'd have to change Elasticsearch's config file), but we would change the behavior of `SearchResults.page_count` and `get_page`.

I'd bet we change it once (as a part of getting it merged) or never, then never again. Based on that I don't think we should bother putting it in `settings.py` (because we'll never override it), but if someone thinks it should be in there I have no moral or philosophical objections.